topostats/io.py

```diff
@@ -131,6 +133,41 @@ def find_images(base_dir: Union[str, Path] = None, file_ext: str = ".spm") -> Li
     return list(base_dir.glob("**/*" + file_ext))
 
 
+def folder_grainstats(output_dir: Union[str, Path], base_dir: Union[str, Path], all_stats_df: pd.DataFrame) -> None:
```

Comment:
Suggestion: rename `folder_grainstats` to `save_folder_grainstats`?

Ignore if you disagree!